Makefile

```diff
@@ -574,6 +573,15 @@ loki-operator-image-cross:
 loki-operator-push: loki-operator-image-cross
 	$(SUDO) $(PUSH_OCI) $(IMAGE_PREFIX)/loki-operator:$(IMAGE_TAG)
 
+#################
+# Documentation #
+#################
+
+documentation-helm-reference-check:
```

Comment:
I wonder if this should be part of the `docs/Makefile` instead.

Reply:
I don't have strong opinions, I'm also fine just leaving it for now.